cloudburst/client/client.py

```diff
@@ -58,7 +58,11 @@ def __init__(self, func_addr, ip, tid=0, local=False):
 
         self.service_addr = 'tcp://' + func_addr + ':%d'
         self.context = zmq.Context(1)
+
         kvs_addr = self._connect()
+        while not kvs_addr:
+            logging.info('Connection timed out, retrying')
```

Comment:
Let's print this out, so that the user knows what's happening.

Reply:
I am doing print and logging in case we wanna inspect the log in the benchmark pods.